syntax = "proto3";
package ampy.metrics.v1;

option csharp_namespace = "Ampy.Metrics.V1";
option java_multiple_files = true;
option java_package = "dev.ampy.metrics.v1";
option objc_class_prefix = "AmpyMetV1";

import "google/protobuf/timestamp.proto";

message Metric {
  string name = 1;
  map<string,string> labels = 2;
  double value = 3;
  google.protobuf.Timestamp at = 4;
}

message MetricBatch {
  repeated Metric metrics = 1;
}
